environment.sh

```diff
@@ -1,17 +1,19 @@
 #!/bin/bash
 ############################################################
-# Help                                                     #
+# Help:                                      #
```

Comment:
I'd just delete this "# Help:..." divider since line 5 is clearly the `Help()` section